packages/gatsby-theme-newrelic/src/components/Link.js

```diff
@@ -43,15 +43,15 @@ const Link = forwardRef(
     `);
 
     const handleExternalLinkClick = useInstrumentedHandler(onClick, {
-      eventName: 'gatsbyTheme',
-      category: 'ExternalLinkClick',
+      eventName: 'externalLinkClick',
+      category: 'GatsbyTheme',
```

Comment:
This is definitely a great change, and the implementation is up for discussion, but I would say the event we are querying in NR1 should not say `gatsbyTheme`, so if we don't include that in the eventName or category it would probably save confusion
```suggestion
      eventName: 'externalLinkClick',
      category: 'LinkClick',
      origin: 'gatsbyTheme'
```